### ISSUE TITLE
Razorgore is unkillable if you reset

### ISSUE THREAD

**Issue by Halyeth** [Duskhaven/Bug-Reports#645]
If you wipe on the attempt killing the eggs and retry after all the eggs are dead Razorgore will stay at one hp and cannot be killed. I was streaming when it happened so I can show it happening live. https://clips.twitch.tv/EnticingArborealEmuStinkyCheese-QqdHicOyvXFvPRIP

**Comment by Feardonald** [comment]
That is correct. Until they fix this, you can remake raid and reset instance to make the eggs respawn.

Also be sure to kill the three NPC's on the platform before you start your MC.

Best Regards
Genjuro

**Comment by harukaze3** [comment]
Until the issue is fixed, you can also open a ticket in game, and if it doesn't get answered in 30 mins, try Genjuro's advice.

> GM Haruka
